staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/webhook_converter.go

```diff
@@ -22,7 +22,7 @@ import (
 	"fmt"
 	"time"
 
-	internal "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
apiextensionsv1 everywhere throughout the PR please.

Reply:
the last commit does it